File: 프로그래머스/2/42578. 의상/의상.js
function solution(clothes) {
    var answer = 0;
return Object.values(clothes.reduce((acc, cur)=>{
       acc[cur[1]] = acc[cur[1]] ? acc[cur[1]]+1 : 1;
       return acc;
   }, {})).reduce((a,b)=> a*(b+1), 1) -1
}

File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    answer = 1
    dic = {c[1]: [] for c in clothes}
    for c in clothes:
        dic[c[1]].append(c[0])
    for k,v in dic.items():
        answer *= (len(v) + 1)
    return answer - 1